packages/uicore/src/components/FormikForm/__tests__/FormikForm.test.tsx

```diff
@@ -573,4 +574,28 @@ describe('<FormInput.KVTagInput />', () => {
 
     await waitFor(() => expect(onSubmit).toBeCalledWith({ tags: ['t1', 't2', 't3', 't4'] }, expect.anything()))
   })
+  test('test getIdentifierFromName() function', () => {
```

Comment:
Pretty comprehensive! Good work!

Reply:
Yes, good work. But, we should also add one test case for NameWithIdentifier component.

Reply:
i think you mean `InputWithIdentifier` component.
<img width="435" alt="image" src="https://github.com/harness/uicore/assets/110456765/eaea8531-08ef-4445-9951-a1a5672789ae">

that will be tested by default from `harness-core-ui` after this change

Reply:
Yeah, I think that's the one. Component should have its own test cases. Ideally, we do not need to test in harness-core-ui. But, anyway that gets tested everywhere along with other things.
If I create new micro-frontend tomorrow, use this component and if somehow flow breaks, how would I know? So, it is important to have tests for all use cases in a dedicated test file of that component.